RELEASES.md

```diff
@@ -19,47 +38,50 @@
       If changes should not be backported, keep them in the _changelogs/head.asciidoc_ file.
     * Patch version: Add new section to existing release notes. ([Sample PR](https://github.com/elastic/apm-server/pull/2064/files))
 
-    Create PR in `main` and backport.
+    Create the changelog PR in `main` and backport to the release branch.
 
-  * Run the [`check_changelogs.py`](script/check_changelogs.py) script to ensure changelogs are synced across branches. This will soon be a PR check.
+  * The [`check_changelogs.py`](script/check_changelogs.py) script is run as a PR check, ensuring that changelog changes are synced across branches.
   * Don't forget to update the "SUPPORTED_VERSIONS" to include a new branch if necessary.
```

Comment:
What are the criteria for update of `SUPPORTED_VERSIONS`? A new branch?

Also, not sure if this is still valid as the `SUPPORTED_VERSIONS` list has no versions for `8.x` and has a `7.x` version without there being no explicit `7.x` branch causing the script to fail.

Reply:
Good catch, I removed this line now.